packages/reload/reload.js

```diff
@@ -223,7 +223,7 @@ Reload._reload = function (options) {
       // with the server if we still have a valid cached copy. This doesn't work
       // when the location contains a hash however, because that wouldn't reload
       // the page and just scroll to the hash location instead.
-      if (window.location.hash) {
+      if (window.location.hash || _.last(window.location.href) === '#') {
```

Comment:
Can you add `api.use('ecmascript-runtime')` to `reload/package.js` and use `window.location.href.endsWith("#")` here? That will trigger another test run, too, which I think should pass this time…